tests/test_examples.py

```diff
@@ -140,3 +142,82 @@ def test_smc_freesolv(smc_free_solv_path):
     # * MAE of ~1.1 kcal/mol: FreeSolv reference calculations
     #   https://www.biorxiv.org/content/10.1101/104281v1.full
     assert mean_abs_err_kcalmol <= 2
+
+
+@pytest.fixture(scope="module")
+def rbfe_edge_list_hif2a_path():
+    def run(results_csv, temp_dir):
+        # expect running this script to write success_rbfe_result_{mol_a_name}_{mol_b_name}.pkl files
+        output_path = str(Path(temp_dir) / "success_rbfe_result*.pkl")
+        assert len(glob(output_path)) == 0
+        config = dict(results_csv=results_csv, **base_config)
+        _ = run_example("rbfe_edge_list.py", get_cli_args(config), cwd=temp_dir)
+        return Path(temp_dir)
+
+    with resources.as_file(resources.files("timemachine.datasets.fep_benchmark.hif2a")) as hif2a_data:
+        base_config = dict(
+            n_frames=2,
+            ligands=hif2a_data / "ligands.sdf",
+            forcefield=DEFAULT_FF,
+            protein=hif2a_data / "5tbm_prepared.pdb",
+            n_gpus=1,
+            seed=2023,
+            n_windows=3,
+        )
+
+        with open(hif2a_data / "results_edges_5ns.csv", "r") as fp:
+            edges_rows = fp.readlines()
+
+        edges_rows_sample = edges_rows[:3]  # keep header and first 2 edges (338 -> 165, 338 -> 215)
+        edges = [("338", "165"), ("338", "215")]
+
+        with temporary_working_dir() as temp_dir:
+            with (Path(temp_dir) / "edges.csv").open("w") as fp:
+                fp.writelines(edges_rows_sample)
+                fp.flush()
+                path = run(fp.name, temp_dir)
+            yield path, base_config, edges
+
+
+def test_rbfe_edge_list_hif2a(rbfe_edge_list_hif2a_path):
```

Comment:
How long does this test take to run?

Reply:
On the order of 10 minutes with using 1 GPU on my local machine.

Reply:
Ok. If you want to make it quicker, you can add n_eq_steps/n_windows as parameters to the script. As a nightly it doesn't matter as much, but this might be quick enough for a unit test.

Reply:
I tried parameterizing `n_eq_steps` and setting to 2 (I had already been limiting `n_windows` to 3, but had initially missed committing that change). The timing is still a bit long for a unit test (~3 minutes), so I'll probably leave this as a nightly for now unless there are any objections.